docs/contributor/testing-plan.md

```diff
@@ -0,0 +1,133 @@
+
+## Appendix B. Quality Evidence Artefacts
+
+This section presents a draft list of quality evidence artifacts that can be created using quality-oriented activities.
+
+| Artefact                     | Links                                                        |
+| ---------------------------- | ------------------------------------------------------------ |
+| Unit test coverage           | ● Unit test coverage expectation document<br />● Unit test coverage report |
+| Functional correctness       | ● The functional/acceptance testing report<br />● Functional testing approach outline |
+| Functional completeness      | ● Coverage analysis of functional aspects by the acceptance test suite |
+| Performance                  | ● Module expected Service Level Agreements (SLAs) <br />● Performance testing strategy and approach<br />● Performance testing report which includes time behaviour and resource utilisation |
+| Maintainability & Modularity | ● Code review code of conduct <br />● Code-style agreement<br />● Project structure description<br />● Modular metrics and expectations |
+| Reusability                  | ● TBD. How reusable is the codebase?                         |
+| Testability                  | ● TBD. How Testable os the codebase?                         |
+| Interoperability             | ● Supported cloud providers list<br />                       |
+| Faultlessness                | ● The report with the trend of the Mean Time Between Failures (MTBF) <br /> |
+| TBD...                       |                                                              |
+
+## Appendix C. Supplementary Documents
+
+This section outlines supplementary documents aimed at improving the quality of the engineering process.
+
+| Aspect               | Document                            | Purpose                                                      |
+| -------------------- | ----------------------------------- | ------------------------------------------------------------ |
+| Software Engineering | Code Style Code of Conduct          | This document defines the coding baseline for the software engineering activities. |
+| Software Engineering | Code Review Code of Conduct         | This document outlines the baseline expectations for the code-review process, including which test suites are mandatory and which are optional, the necessity of system diagrams for architectural changes, and more... |
+| Software Engineering | Project Structure                   | This document includes the agreement on the architecture and structure of the project. |
+| Monitoring           | The metrics generated by the module | This document outlines the observation-critical metrics to be used for monitoring the module. |
+| Integration Testing  | External dependencies isolation     | This document outlines the approach to replace third-party dependencies with controllable test-doubles. |
```

Comment:
If these are existing docs, please provide the links.

Reply:
The documents don't exist or are not yet adequately defined.